src/components/shared/Avatar.tsx

```diff
@@ -0,0 +1,72 @@
+import clsx from "clsx";
+import type { VFC } from "react";
+
+import { Button } from "./Button";
+
+type CommonType = {
+  alt: string;
+  className?: string;
+  size?: "small" | "medium" | "large" | "extralarge";
+  onClick?: (e: React.MouseEvent<HTMLElement>) => void;
+};
+
+type ImgType = CommonType & {
+  src: string;
+};
+
+type ButtonType = CommonType;
+
+// props の型がimage型かbutton型かを判断する関数を定義する。
+// 画像ファイルがある場合にimgタブで表示する
+const isImg = (props: ImgType | ButtonType): props is ImgType => {
+  return "src" in props && props.src !== "";
```

Comment:
おそらくこれだけでいけますね！🧐
`sample.tsx`の`src=""`は消す必要があります！

```suggestion
  return "src" in props;
```

Reply:
削除すると下記のようになってしまいました。こちらは修正せずに進めます。
![image](https://user-images.githubusercontent.com/75880059/112983222-60cc7280-9198-11eb-8990-128c3177d1bf.png)

Reply:
sample.tsxのsrc=""は消されましたか？
そちらを消すとうまく動くかと思います！